### PR TITLE
Changelog for 9.19.0 release

### PR DESCRIPTION
**Pull request by wuweiweiwu**
- [x] smoke tests all look good on Chrome, IE11, Edge, Safari, Firefox
- [x] `yarn test:ci` passes
- [x] `flow` and `lint` passes
- [x] changelog updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,24 @@
 Changelog
 ------------
 
-##### NEXT
+##### 9.19.0
+
 * Replaced `componentWillMount`, `componentWillReceiveProps`, and `componentWillUpdate` with async-safe lifecycles in advance of React 16.4 deprecation warnings. Added [react-lifecycles-compat](https://github.com/reactjs/react-lifecycles-compat) as a dependency, to ensure backwards compatibility.
+* Public flow interface for `CellMeasurer`. ([diogofcunha](https://github.com/diogofcunha)) - [#1058](https://github.com/bvaughn/react-virtualized/pull/1058)
```
Comment (TrySound) [inline CHANGELOG.md]: Did you mean `publish`?

Comment (TrySound) [reply CHANGELOG.md]: Opps, nevermind.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,24 @@
 Changelog
 ------------
 
-##### NEXT
+##### 9.19.0
+
 * Replaced `componentWillMount`, `componentWillReceiveProps`, and `componentWillUpdate` with async-safe lifecycles in advance of React 16.4 deprecation warnings. Added [react-lifecycles-compat](https://github.com/reactjs/react-lifecycles-compat) as a dependency, to ensure backwards compatibility.
```
Comment (bvaughn) [inline CHANGELOG.md]: I realize you didn't write this line- (I did)- but can we change "_React 16.4 deprecation warnings_" to "_React 16.x deprecation warnings_". It looks likely that we will not be enabling them with 16.4, but in a subsequent minor release.

Comment (wuweiweiwu) [reply CHANGELOG.md]: I see. Will update.

**Pull request merged**